Use scummvm-ios7-libs-v3 (#4856)
scummvm/scummvm: doc/docportal/other_platforms/ios.rst

    mkdir build

> Move the contents of the ``scummvm-ios7-libs-v3`` folder you extracted earlier into the ``build`` directory, using either the Finder, or the command line as follows:

Reviewer: ```suggestion
Move the ``frameworks`` folder you extracted earlier from the ``scummvm-ios7-libs-v3.zip`` file into the ``build`` directory, using either the Finder, or the command line as follows:
```

.. code-block::

>     cp -r ~/Downloads/scummvm-ios7-libs-v3/* ~/build/

Reviewer: The new package is extracted as `frameworks` for me and not `scummvm-ios7-libs-v3/frameworks`.
So the instruction becomes: `cp -r ~/Downloads/frameworks ~/build/`
Or maybe we should suggest `mv ~/Downloads/frameworks ~/build/`
